app/src/Components/ListPropertyPages/Page2.js

```diff
@@ -86,7 +91,7 @@ const Page2 = ({onNext,onPrevious}) =>{
     onRequestClose={closeModal}
     style={{
         overlay: {
-         zIndex: 999,
```

Comment:
Why is this set so high? We should be setting zIndex incrementally from 0 not assigning a value near maxima.